### ISSUE TITLE
Bug: Check \ escaping works for player detections

### ISSUE THREAD

**Issue by dd84ai** [darklab8/darklab_darkbot#36]
`. player friend tags add R\-` is reported not finding players properly.
Some escaping problems with \ thing are expected.
Create test char and verify how it all works and fix if necessary

Verify info returned by player api. Potentially corrupted at this point too

**Comment by dd84ai** [comment]
`R\Nickname\-Tester` entered game as `R\\Nickname\\-Tester`

**Comment by dd84ai** [comment]
![Screenshot_20241117_184049](https://github.com/user-attachments/assets/1221e54b-b99f-4f3c-b2bb-4c6ba8a30fa3)
i am able to monitor the player with such nickname

**Comment by dd84ai** [comment]
So... player `R\Nickname\-Tester` gets scrapped from Disco appi as
```
	// player.Player
	// Visible as
	// Time = "4m"
	// Name = "R\\Nickname\\-Tester"
	// System = "Pennsylvania"
	// Region = "Liberty Space"
```

Discord command `; player friend add R\N, gets as R\\N`

In both cases Disco api and Darkbot config escape data in same way? What is the problem?
Problem is the confusing data `; conf` command shown.
and responses from player tags added may be.

The data should be escaped with three quotations marks, then everything will be ok

**Comment by dd84ai** [comment]
```
		fmt.Println(tags, err)
		fmt.Println(fmt.Sprintf("%#v", tags))
```
prints as
```
[Nickname\\\\-Tester Nickname\\-Tester R\N Smth\S] <nil>
[]types.Tag{"Nickname\\\\\\\\-Tester", "Nickname\\\\-Tester", "R\\N", "Smth\\S"}
```
Extra confusion here that we are printing values not as they are trully are for `conf` command
standard Println escapes stuff :/

**Comment by dd84ai** [comment]
![Image](https://github.com/user-attachments/assets/24123347-08ab-42b2-a36e-c005be1518b1)
Added tag adding command to show how the thing is added

**Comment by dd84ai** [comment]
![Image](https://github.com/user-attachments/assets/02793f84-97d9-463e-a9f3-e98bb9475e98)
also made sure `; conf` command prints the tags and other settings exactly as they will be used for rendered table.

TLDR: aligned all data to the same level of escaping to remove confusion

**Comment by dd84ai** [comment]
So... as end result, players with difficult tags should do next thing.

`. player region add Liberty` add regions with players to see how they are added into darkbot, then with seeing this info just add
`. player friend add R\-` for example, no extra escaping is needed. You will see response now how the tag was added into darkbot 1 to 1 correctly
and u will be able to verify in advance that all is defined correctly.

That is i think the best solution within this situation. Adjusted darkbot version will be deployed for more useful/precise information regarding configuration

**Comment by dd84ai** [comment]
Closing with anouncement in https://discoverygc.com/forums/showthread.php?tid=188040&pid=2351047#pid2351047
